zenoh/src/sample.rs

```diff
@@ -33,6 +33,7 @@ pub enum Locality {
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub(crate) enum Locality {
     SessionLocal,
+    Remote,
```

Comment:
Is this an intended change?

Reply:
Yes, otherwise a library importing zenoh `feature = "unstable)` won't compile.